speech-dispatcher.socket

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Speech Dispatcher Socket
+
+[Socket]
```

Comment:
maybe a good idea to set Priority=

Reply:
Hmm I don't know if that should have low, medium or high priority.